packaging/pkg_helpers.bash

```diff
@@ -172,8 +172,8 @@ setup_pip_pytorch_version() {
     fi
   else
     pip_install "torch==$PYTORCH_VERSION$PYTORCH_VERSION_SUFFIX" \
-      -f https://download.pytorch.org/whl/torch_stable.html \
-      -f https://download.pytorch.org/whl/nightly/torch_nightly.html
+      -f "https://download.pytorch.org/whl/${UPLOAD_CHANNEL}/torch_${UPLOAD_CHANNEL}.html" \
+      -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
@seemethere I am not sure what is the right order between stable and `torch_nightly/torch_test`. Could you advise?